### ISSUE TITLE
Admin User login requires name field in request body

### ISSUE THREAD

**Issue by saiteja310** [stackinflow/focial-backend#32]
validateRegisterFields is being called instead of validateLoginFields in Admin login route

**Comment by fayaz07** [comment]
yeah my bad, initially it was same. I mean we used to just get email and password for both login and register routes, but eventually in the end we added `name` param in `register` route. I just forgot to change that.

**Comment by saiteja310** [comment]
> yeah my bad, initially it was same. I mean we used to just get email and password for both login and register routes, but eventually in the end we added `name` param in `register` route. I just forgot to change that.

Raising a pr for that, found as part of testing the swagger apis. 👍
